jupyter_elfinder/views.py

```diff
@@ -80,8 +80,9 @@ def connector(request: Request) -> Response:
         "url": request.registry.settings["files_url"],
         "base_url": request.registry.settings["base_url"],
         "upload_max_size": 100 * 1024 * 1024 * 1024,  # 100GB
-        "debug": True,
         "tmb_dir": request.registry.settings.get("jupyter_elfinder_thumbnail_dir"),
+        "send_file_path": request.registry.settings["send_file_path"],
+        "debug": True,
```

Comment:
Should we add a debug option in the cli arguments? Can be another PR.

Reply:
Good idea. Let's do that in another PR.